package/MDAnalysis/coordinates/LAMMPS.py

```diff
@@ -458,19 +458,53 @@ class DumpReader(base.ReaderBase):
     "unwrapped" (xu,yu,zu) and "scaled_unwrapped" (xsu,ysu,zsu) coordinate
     conventions (see https://docs.lammps.org/dump.html for more details).
     If `lammps_coordinate_convention='auto'` (default),
-    one will be guessed. Guessing checks whether the coordinates fit each convention in the order "unscaled",
-    "scaled", "unwrapped", "scaled_unwrapped" and whichever set of coordinates
-    is detected first will be used. If coordinates are given in the scaled
-    coordinate convention (xs,ys,zs) or scaled unwrapped coordinate convention
-    (xsu,ysu,zsu) they will automatically be converted from their
-    scaled/fractional representation to their real values.
+    one will be guessed. Guessing checks whether the coordinates fit each
+    convention in the order "unscaled", "scaled", "unwrapped",
+    "scaled_unwrapped" and whichever set of coordinates is detected first will
+    be used. If coordinates are given in the scaled coordinate convention
+    (xs,ys,zs) or scaled unwrapped coordinate convention (xsu,ysu,zsu) they
+    will automatically be converted from their scaled/fractional representation
+    to their real values.
 
     Supports both orthogonal and triclinic simulation box dimensions (for more
     details see https://docs.lammps.org/Howto_triclinic.html). In either case,
     MDAnalysis will always use ``(*A*, *B*, *C*, *alpha*, *beta*, *gamma*)``
     to represent the unit cell. Lengths *A*, *B*, *C* are in the MDAnalysis
     length unit (Å), and angles are in degrees.
 
+    By using the keyword `additional_columns`, you can specify arbitrary data
+    to be read alongside the coordinates. If specified, the keyword expects a
+    list of the names of the columns that you want to have read. The results
+    of the parsing are saved to the time step `data` dictionary alongside the
+    name of the data column. For instance, if you have time-dependent charges
+    saved in a LAMMPS dump such as
+
+    .. code-block:: python
+
+        ITEM: ATOMS id x y z q l
+        1 2.84 8.17 -25 0.00258855 1.1
+        2 7.1 8.17 -25 6.91952e-05 1.2
+
+    Then you may parse the additional columns `q` and `l` via.
+
+    .. code-block:: python
+
+        u = mda.Universe('path_to_data', 'path_to_lammpsdump',
+                         additional_columns=['q', 'l'])
+
+    The additional data is then available for each time step via
+    (as the value of the `data` dictionary, sorted by the ids of the atoms).
+
+    .. code-block:: python
+
+        for ts in u.trajectory:
+            charges = ts.data['q'] # Access the additional data, sorted by the id
+            ls = ts.data['l']
+        ...
+
+    .. versionchanged:: 2.2.0
+       Reading of arbitrary, additional columns is now supported.
+       (Issue `#3608 <https://github.com/MDAnalysis/mdanalysis/pull/3608>`__)
```

Comment:
No need for the full link IMO

Reply:
Address this, just `Issue #3608` should be fine.

Reply:
hopefully fine now :)